### DISCUSSION TITLE
Logowanie kontem Azure AD do Windows Azure VM.

### DISCUSSION THREAD

**Discussion by MirekTkaczyk** [AzureCommunityPL/group#377] [Pomoc] [answered]
Hej. Mam problem z połączeniem do VM z Windows Server 2019 kontem AzureAD\user bez uprawnień admina. Łączę się kontem AzureAD\user z dodaną role Machine Administrator Login, lub kontem lokalnym. Ten sam AzureAD\user z role Virtal Machine User Login dostaje  “The requested session access is denied”.
Dodany Extension AADLoginFor Windows, dodane CAL per User. Logowanie bez MFA.
AzureAD\user dodany do Remote desktop users.
RDP z: enablecredsspsupport:i:0, authentication level:i:2
Coś pominąłem?

**Comment by smereczynski** [comment]
To jest domena czy grupa robocza?

**Comment by MirekTkaczyk** [comment, marked as answer]
Znalazłem rozwiązanie. Chciałem logować się uprawnieniami usera a w .rdp sesja admina: administrative session:i:1
Zmiana na 0 rozwiązuje problem
